web/client/selectors/maptype.js

```diff
@@ -22,8 +22,12 @@ const mapTypeSelector = (state) => state && state.maptype && state.maptype.mapTy
  * @return {boolean}
  */
 const isCesium = state => mapTypeSelector(state) === "cesium";
+const isLeaflet = state => mapTypeSelector(state) === "leaflet";
```

Comment:
it is not used anymore, but i think it can be useful in the future. But i can understand your point.